api/edge_api/identities/serializers.py

```diff
@@ -20,6 +22,28 @@
 engine_multi_fs_value_schema = MultivariateFeatureStateValueSchema()
 
 
+class EdgeIdentitySerializer(serializers.ModelSerializer):
+    identity_uuid = serializers.CharField(required=False)
+
+    class Meta:
+        model = Identity
```

Comment:
The above is kind of validated by the fact that this line is only here to stop it from erroring, right?